pkg/porter/testdata/cache/887e7e65e39277f8744bd00278760b06/cnab/bundle.json

```diff
@@ -7,7 +7,7 @@
     "description": "",
     "images": null,
     "invocationImages": [{
-        "digest": "sha256:f858bc025ad34099fe67ebe6152e03b4c91b34cc7a77d1aa10aaf1dc1389c2c2",
+        "contentDigest": "sha256:f858bc025ad34099fe67ebe6152e03b4c91b34cc7a77d1aa10aaf1dc1389c2c2",
```

Comment:
Did you see this same mistake anywhere else in our repo? Would you mind taking a quick look?

Reply:
Ooh, yea, there are def other spots.  In fact, for the `images`/ImageMap part of a manifest, the yaml tag is still `digest`, so there will be quite a few changes.  Do we want to add these as an additional commit to this PR or as a follow-up?

Reply:
Follow-up please! 🙇‍♀️